sdk/tests/conformance2/glsl3/shader-with-invalid-characters.html

```diff
@@ -49,9 +49,36 @@
     gl_Position = vec4(1.0, 0.0, 0.0, 1.0);
 }
 </script>
+<script id="vertexShaderBackslashValid" type="text/something-not-javascript">
+void main()
+{
+    gl_Position = \
+        vec4(1.0, 0.0, 0.0, 1.0);
+}
+</script>
+<script id="vertexShaderBackslashInvalid" type="text/something-not-javascript">
+void main()
+{
+    gl_Position = \ vec4(1.0, 0.0, 0.0, 1.0);
```

Comment:
I think a more meaningful test is after \ there are a few white space
that could be common mistake to make

In this situation, it's better you put source directly as a string in the source instead of a html element - to make sure lint tools won't get rid of the whitespace in the end of a line